charts/astronomer/templates/commander/commander-role.yaml

```diff
@@ -29,20 +29,26 @@ metadata:
     release: {{ $.Release.Name }}
     chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
     heritage: {{ $.Release.Service }}
-  {{- if $.Values.global.enableArgoCDAnnotation }}
+
   annotations:
+    {{- if not $.Values.global.features.namespacePools.namespaces.create }}
+    helm.sh/resource-policy: keep
```

Comment:
Why are we adding this? Won't this orphan this object? https://helm.sh/docs/howto/charts_tips_and_tricks/#tell-helm-not-to-uninstall-a-resource

Reply:
So this is done to make sure already created roles and role bindings are not deleted that is why this keep flag is introduced. @danielhoherd

Reply:
When we do this, we lose all control of helm managing these resources. Is that really what we want? CC @pgvishnuram 

> However, this resource becomes orphaned. Helm will no longer manage it in any way. This can lead to problems if using helm install --replace on a release that has already been uninstalled, but has kept resources.